geoalchemy/base.py

```diff
@@ -89,7 +90,6 @@ class WKBSpatialElement(SpatialElement, Function):
     """
 
     def __init__(self, desc, srid=4326, geometry_type='GEOMETRY'):
-        assert isinstance(desc, (basestring, buffer))
```

Comment:
there's no Python 3 equivalent of `buffer` so I figured we can just delete this assertion